components/ProjectForm/index.js

```diff
@@ -0,0 +1,127 @@
+import { useForm } from "react-hook-form";
+import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+const StyledErrorMessage = styled.p`
+  color: red;
+`;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  align-items: center;
+`;
+
+export default function ProjectForm({ projects, onAddProject }) {
+  const router = useRouter();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset,
+  } = useForm();
+
+  const onSubmit = (data) => {
+    onAddProject(data);
+    console.log(projects);
+    reset();
+    router.push("/");
+  };
+
+  return (
+    <>
+      <StyledForm onSubmit={handleSubmit(onSubmit)}>
+        <h2>Create new project</h2>
+        <label>
+          Project title
+          <input
+            {...register("title", { required: "Title is required" })}
+            placeholder="Title"
+          />
+          <StyledErrorMessage>{errors.title?.message}</StyledErrorMessage>
+        </label>
+
+        <label>
+          Image
+          <input {...register("image")} placeholder="Select Image" />
+        </label>
+
+        <label>
+          Description
+          <textarea
+            {...register("description", {
+              required: "Description is required",
+            })}
+            placeholder="Description"
+          />
+          <StyledErrorMessage>{errors.description?.message}</StyledErrorMessage>
+        </label>
+        <div>
+          <label>
+            Duration
+            <select {...register("duration")} defaultValue={"select..."}>
+              <option value="select...">select...</option>
+              <option value="short">short</option>
+              <option value="medium">medium</option>
+              <option value="long">long</option>
+            </select>
+          </label>
+          &nbsp;
```

Comment:
Instead of creating a space by adding this I would prefer you just style the `div` element. :)